test/unit/remote/change.js

```diff
@@ -18,4 +18,30 @@ describe('remote change sort', () => {
     remoteChange.sort(a)
     a.should.deepEqual([parent, child])
   })
+
+  describe('sorts deleted before created for the same path', () => {
+    const deleted = {
+      doc: { path: 'parent/file' },
+      type: 'FileDeletion',
+      was: { path: 'parent/file' }
```

Comment:
`RemoteFileDeletion` is not supposed to have a `was` (and the test still pass without it).

Reply:
I thought I'd seen it in logs :thinking:. I just wanted to reproduce that situation.
I'll remove the `was` then.

Reply:
I'm almost sure for this one: the property is not part of the type and when logging `FileDeletion` on some scenario it has a `doc` but not a `was`.